src/Microsoft.Extensions.FileProviders.Embedded/EmbeddedFileProvider.cs

```diff
@@ -49,8 +49,15 @@ public EmbeddedFileProvider(Assembly assembly, string baseNamespace)
 
             _baseNamespace = string.IsNullOrEmpty(baseNamespace) ? string.Empty : baseNamespace + ".";
             _assembly = assembly;
-            // REVIEW: Does this even make sense?
-            _lastModified = DateTimeOffset.MaxValue;
+
+            _lastModified = DateTimeOffset.UtcNow;
+
+#if NETSTANDARD1_5
+            if (_assembly.Location != null)
+            {
+                _lastModified = File.GetLastWriteTimeUtc(_assembly.Location);
```

Comment:
This can throw a lot of exceptions that we may be better off suppressing.
https://msdn.microsoft.com/en-us/library/system.io.file.getlastwritetimeutc(v=vs.110).aspx